test/tests/functional/pbs_accumulate_resc_used.py

```diff
@@ -138,12 +138,9 @@ def setUp(self):
         self.assertEqual(rc, 0)
 
         # Ensure the new resource is seen by all moms.
-        self.momA.log_match(
-            "resourcedef;copy hook-related file", max_attempts=3)
-        self.momB.log_match(
-            "resourcedef;copy hook-related file", max_attempts=3)
-        self.momC.log_match(
-            "resourcedef;copy hook-related file", max_attempts=3)
+        self.momA.log_match("resourcedef;copy hook-related file")
+        self.momB.log_match("resourcedef;copy hook-related file")
+        self.momC.log_match("resourcedef;copy hook-related file")
```

Comment:
Please create a list of self.momA, self.momB, and self.momC and do a for loop:
```suggestion
        moms = [self.momA, self.momB, self.momC]
        for m in mom:
            m.log_match("resourcedef;copy hook-related file")
```
Here and below